docker/Dockerfile

```diff
@@ -1,63 +1,44 @@
-FROM  python:3.9-alpine@sha256:e80214a705236091ee3821a7e512e80bd3337b50a95392a36b9a40b8fc0ea183  as  build
+FROM alpine/git as git-clone
 
-LABEL "maintainer"="FIWARE Foundation e.V."
-LABEL "description"="An Internet of Things Agent for the RDF DataQube Turtle format. This IoT Agent is designed to be a bridge between ISOXML/ADAPT and the ETSI NGSI-LD interface of a FIWARE Context Broker."
-LABEL "name"="iotagent-turtle"
-LABEL "summary"="IoT Agent for the DataQube representation in RDF Turtle format used in statistical environments"
+ARG PROJECT=flopezag
+ARG COMPONENT=IoTAgent-Turtle
+ARG VERSION=develop
```

Comment:
This should be BRANCH not VERSION

Reply:
Reset the labels in the final image for the docker